docs/includes/usage-examples/UpdateManyTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use MongoDB\Laravel\Tests\TestCase;
+
+class UpdateManyTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testUpdateMany(): void
+    {
+        require_once __DIR__ . '/Movie.php';
+
+        Movie::truncate();
+        Movie::insert([
+            [
+                'title' => 'Hollywood',
+                'imdb' => [
+                    'rating' => 9.1,
+                    'votes' => 511,
+                ],
+            ],
+            [
+                'title' => 'The Shawshank Redemption',
+                'imdb' => [
+                    'rating' => 9.3,
+                    'votes' => 1513145,
+                ],
+            ],
+        ]);
+
+        // begin-update-many
+        $updates = Movie::where('imdb.rating', '>', 9.0)
+            ->update(['$set' => ['acclaimed' => true]]);
```

Comment:
`$set` is optional, you can pass the values directly for compatibility with SQL Eloquent.

```suggestion
            ->update(['acclaimed' => true]);
```

The operator `$set` is [added automatically](https://github.com/mongodb/laravel-mongodb/blob/95c5b4fac69c2bd5ea01367e62d14dd5278e7513/src/Query/Builder.php#L711-L721) to the command when the values doesn't contain any $-prefixed name. Example: https://github.com/mongodb/laravel-mongodb/blob/95c5b4fac69c2bd5ea01367e62d14dd5278e7513/tests/QueryTest.php#L578

Reply:
Good to know, thanks!